engine/src/flutter/lib/ui/window.dart

```diff
@@ -378,8 +379,8 @@ class FlutterView {
     _render(
       viewId,
       scene as _NativeScene,
-      size?.width ?? physicalSize.width,
-      size?.height ?? physicalSize.height,
+      size?.width ?? _viewConfiguration.viewConstraints.maxWidth,
+      size?.height ?? _viewConfiguration.viewConstraints.maxHeight,
```

Comment:
In my testing this didn't need to change.

Reply:
interesting. let me try that